UltraStar Play/Assets/Scenes/Sing/SingSceneController.cs

```diff
@@ -330,6 +334,10 @@ public void FinishScene()
         // Open the singing results scene.
         SingingResultsSceneData singingResultsSceneData = new SingingResultsSceneData();
         singingResultsSceneData.SongMeta = SongMeta;
+		
+		//Get the stats manager and the stats object
+        var stats = StatsManager.Instance.Statistics;
```

Comment:
Here as well: The indentation seems to use tabs instead of spaces sometimes. Please check that your editor always uses spaces for indentation.